advocacy_docs/supported-open-source/patroni/installing_on_rhel.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Installing Patroni on RedHat compatible systems'
+navTitle: 'Installing Patroni on RHEL'
+description: "A walkthrough on how to install Patroni with PostgreSQL or EPAS on RedHat Enterprise Linux family systems"
+---
+
+This walks you through the various steps to create an highly available demo cluster using EDB Postgres Advanced Server version 14 and PostgreSQL 14 on RedHat Enterprise Linux 8. The two database nodes in _Streaming Replication_ are pg-patroni1 and pg-patroni2.
+
+## EDB Postgres Advanced Server
+
+On both pg-patroni1 and pg-patroni2 hosts, install EPAS using the following commands:
+
+```bash
+sudo dnf -y install https://yum.enterprisedb.com/edbrepos/edb-repo-latest.noarch.rpm
+
+#  Replace 'USERNAME' and 'PASSWORD' below with
+#  your username and password for the EDB repository:
+sudo sed -i "s@<username>:<password>@USERNAME:PASSWORD@" /etc/yum.repos.d/edb.repo
+
+sudo dnf -y install epel-release
+sudo dnf config-manager --set-enabled PowerTools
```

Comment:
I don't know if this affects things later or not, but on a new rocky linux 8 box I get:

[vagrant@db1 ~]$ sudo dnf config-manager --set-enabled PowerTools
Error: No matching repo to modify: PowerTools.

Reply:
Nice catch! It's `powertools` not `PowerTools`

```
[root@Centos8 ~]# dnf config-manager --set-enabled PowerTools
Error: No matching repo to modify: PowerTools.
[root@Centos8 ~]# dnf config-manager --set-enabled powertools
[root@Centos8 ~]#
```

Reply:
Fixed in 4d2dd18776747591ca560e81e0b33a3da4c462eb